doc/sphinx-guides/source/installation/config.rst

```diff
@@ -124,7 +124,7 @@ Once you have your DOI or Handle account credentials and a namespace, configure
 Configuring Dataverse for DOIs
 ++++++++++++++++++++++++++++++
 
-Out of the box, Dataverse is configured for DOIs. Here are the configuration options for DOIs:
+Out of the box, Dataverse is configured to register DOIs for each dataverse, dataset, and file. Here are the configuration options for DOIs:
```

Comment:
"Out of the box, Dataverse is configured to register DOIs" is probably too strong a statement.

"Out of the box, Dataverse is configured to register fake DOIs" used to be true when EZID was used.

"Out of the box, Dataverse is configured for fake DOIs but you must obtain credentials as explained above" would be accurate, I guess.